util/serial/src/lib.rs

```diff
@@ -55,15 +56,15 @@ pub fn serialize<T: ?Sized>(value: &T) -> Result<Vec<u8>, encode::Error>
 where
     T: serde::ser::Serialize + Sized,
 {
-    Ok(serde_cbor::to_vec(value)?)
+    Ok(cbor::to_vec(value)?)
```

Comment:
This method's comment comment is stale, would you mind updating it while we're here?